packages/react-aria-components/src/Collection.tsx

```diff
@@ -515,6 +519,10 @@ export class Document<T, C extends BaseCollection<T> = BaseCollection<T>> extend
     this.collectionMutated = true;
   }
 
+  get isConnected() {
+    return true;
```

Comment:
shouldn't this be set to true in addNode and false in removeNode?

Reply:
This one is for the `Document`, which is always "connected" to itself. The other nodes recursively check upward through their ancestry.

Reply:
ah, got it. missed that this was the Document itself